packages/xml-views-validation/src/validators/attributes/none-unique-id.ts

```diff
@@ -0,0 +1,79 @@
+import { clone, flatMap, map, omitBy } from "lodash";
+import {
+  accept,
+  XMLAstVisitor,
+  XMLAttribute,
+  XMLDocument,
+  XMLToken,
+} from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { NoneUniqueIDIssue } from "../../../api";
+
+export function validateUniqueID(
+  xmlDoc: XMLDocument,
+  model: UI5SemanticModel
+): NoneUniqueIDIssue[] {
+  const idCollector = new IdsCollectorVisitor();
+  accept(xmlDoc, idCollector);
+  const idsToXMLElements = idCollector.idsToXMLElements;
+
+  const duplicatedIdsRecords = omitBy(idsToXMLElements, (_) => _.length > 1);
+  const issues: NoneUniqueIDIssue[] = flatMap(
+    duplicatedIdsRecords,
+    (currDupAttribsPerID, id) => {
+      return map(currDupAttribsPerID, (currDupAttrib, currIdx) => {
+        const currDuplicatedIdValToken = currDupAttrib.syntax.value;
+        const otherDuplicatedIdsAttribs = clone(currDupAttribsPerID).splice(
+          currIdx
+        );
+
+        return {
+          kind: "NoneUniqueIDIssue",
+          message: `None Unique ID value: "${id}" found.`,
```

Comment:
The old error message was: `The control ID is the same as another control ID in the view. Please enter a unique ID.`
Maybe we shouldn't write "control" (it could be an element) and maybe we should drop the "please" but could you change the message to be similar?

Reply:
I feel the previous message was too long and verbose
and the new message will include "related information" which will add even more lines to the error (verbosity).

- The comment how to fix seems redundant, it should be self evident... e.g see wrong use of const in JetBrains:
  
  ![image](https://user-images.githubusercontent.com/4233843/83352148-4b0cd780-a352-11ea-8e87-38fdd6dbf23c.png)

I'd rather let UX review all these strings at once, while only treating the old implementation as a guideline only.